content/docs/engine-dev/tools/project-manager/source-code-map.md

```diff
@@ -26,6 +30,22 @@ This contains brief summaries of various class's functionality and role. If avai
 |ScreenDefs | [header](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreenDefs.h) | Contains definitions for `ProjectManagerScreen` enum, which describes all possible types of screens in Project Manager. It also defines a hash function, and a mapping to appropriate string equivalents for each enum.|
 |[ScreenWidget](#screenwidget) | [header](https://github.com/o3de/o3de/blob/development/Code/Tools/ProjectManager/Source/ScreenWidget.h) | The parent class to all screens in Project Manager. It contains all necessary stubs for screen management and transition. `ScreensCtrl` is defined in terms of `ScreenWidget`, so that all transition logic is polymorphic.|
 
+[Back to Overview](#overview)
+
+### Data Classes
```

Comment:
I would think you'd want to maintain a hierarchy, and group the table listing all the classes right above the classes themselves. So for example, I would think this table listing all the Data Classes should appear after all the Startup and Screen Management classes. It's also a little confusing that everything is at the same `###` header level. Thoughts?

Reply:
Perhaps it's time to split this topic into subtopic pages?

Reply:
so to clarify I should have it mapped such that each section has its own table of contents and class descriptions? 

So One section would be for startup and screen management, and another for data classes in this case. If so, then I agree that makes sense. 

Regarding header levels, I'm assuming you are referring to how the table of contents and class descriptions have the same header level? If so, I can work on updating that so the intended nesting is clearer.

Reply:
missed your comment about splitting it into sub-pages. I suspect you're correct. The current file is getting very large, and I have only scratched the surface of the codebase for the project manager. I'll look at reorganizing it into sub-pages.